### ISSUE TITLE
With cliquet-fxa, Kinto 1.6.0 doesn't starts with uwsgi because settings['project_name '] is not defined.

### ISSUE THREAD

**Issue by Natim** [Kinto/kinto#213]
```
  File "app.wsgi", line 23, in <module>
    application = main(config.items('DEFAULT'), **dict(config.items('app:main')))
  File "/home/ubuntu/venvs/kinto/local/lib/python2.7/site-packages/kinto/__init__.py", line 33, in main
    config = Configurator(settings=settings, root_factory=RouteFactory)
  File "/home/ubuntu/venvs/kinto/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 301, in __init__
    exceptionresponse_view=exceptionresponse_view,
  File "/home/ubuntu/venvs/kinto/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 412, in setup_registry
    self.include(inc)
  File "/home/ubuntu/venvs/kinto/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 755, in include
    c(configurator)
  File "/home/ubuntu/venvs/kinto/local/lib/python2.7/site-packages/cliquet_fxa/__init__.py", line 26, in includeme
    cliquet.load_default_settings(config, DEFAULT_SETTINGS)
  File "/home/ubuntu/venvs/kinto/local/lib/python2.7/site-packages/cliquet/initialization.py", line 376, in load_default_settings
    project_name = settings['project_name']
KeyError: 'project_name'
```

**Comment by Natim** [comment]
This is only the case when using cliquet_fxa with multiauth.

**Comment by Natim** [comment]
Fixed with cliquet-fxa 1.3.2
